types/session_cost.yaml

```diff
@@ -1,5 +1,62 @@
 description: Types to provide the session price
 types:
+  IdlePrice:
```

Comment:
i'm a little confused - from what i understand we refer to idle to the state when the station has no customer. What does idle mean in this context?

Reply:
For more info, you can read the California Pricing Requirements.
https://openchargealliance.org/whitepapers/ <- to be downloaded there. 

Idle is when a car is attached but currently not charging. For example because the battery is already full. There are charging stations where you have to pay something when you are not charging (I mainly found them in the field with DC chargers, but also AC chargers sometimes have this).

Reply:
cool understood - maybe you can link the req in the description

Reply:
Ah yes the link is already in the issue:
https://github.com/EVerest/libocpp/issues/376